smoke-tests-otel-starter/spring-boot-3/src/test/java/io/opentelemetry/spring/smoketest/GraalVmNativeKafkaSpringStarterSmokeTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.spring.smoketest;
+
+import org.junit.jupiter.api.condition.EnabledInNativeImage;
+import org.springframework.boot.test.context.SpringBootTest;
+
+/**
+ * GraalVM native image doesn't support Testcontainers in our case, so the docker container is
+ * started manually before running the tests.
+ *
+ * <p>In other cases, it does work, e.g. <a
+ * href="https://info.michael-simons.eu/2023/10/25/run-your-integration-tests-against-testcontainers-with-graalvm-native-image/">here</a>,
+ * it's not yet clear why it doesn't work in our case.
+ *
+ * <p>In CI, this is done in reusable-native-tests.yml. If you want to run the tests locally, you
+ * need to start the container manually: see .github/workflows/reusable-native-tests.yml for the
+ * command.
+ */
+@SpringBootTest(
+    classes = {
+      OtelSpringStarterSmokeTestApplication.class,
+      SpringSmokeOtelConfiguration.class,
+      AbstractKafkaSpringStarterSmokeTest.KafkaConfig.class
+    },
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@EnabledInNativeImage // see JvmMongodbSpringStarterSmokeTest for the JVM test
+@EnabledInGithubActions
```

Comment:
This test could be executed locally with Docker compose and by setting an environment variable.

The `EnabledInGithubActions` annotation would suggest that the test could only be executed with a Github action.

So, the usage of `@EnabledIfEnvironmentVariable` here may be more appropriate.

Reply:
ok, I'll rename it to `@RequiresDockerCompose`

Reply:
> ok, I'll rename it to `@RequiresDockerCompose`

It would be misleading because the contributor won't directly know that an environment variable has to be set.

Reply:
just look at the definition of the attribute :smile:

Reply:
What do you think about `@RequiresDockerComposeEnvVariable`?